### DISCUSSION TITLE
Show repo config

### DISCUSSION THREAD

**Discussion by dimejo** [rustic-rs/rustic#381] [Q&A] [answered]
I found some options to set repository configuration (compression, packsize etc.) but couldn't find a command to print/show the current configuration. Did I miss something?

**Comment by aawsome** [comment, marked as answer]
So far there is only `rustic cat config`...

**Reply by dimejo** [reply]
Thanks, that will do for now.

**Comment by aawsome** [comment]
But you are right, there is some command missing to show what configuration actually *is used* - the `cat config` command only shows the raw json, but there are defaults for missing values and some values might profit from explanation...
But at least you can use `rustic cat config` to see what effects the `config` command does have.

**Comment by aawsome** [comment]
(deleted - wrong topic)
